### PR TITLE
Move all tests into github actions

### PR DESCRIPTION
**Pull request by tgodzik**
Previously, we would run tests on Travis, Azure and Github Actions. Now I moved all tests to run on github actions, since they seem more stable now there.

### CODE REVIEW

File: bin/test.sh [hunk 1]
```diff
@@ -1,19 +1,10 @@
...
 
-[ -d "$bloopDir" ] && rm -rf "$bloopDir"
-mkdir "$bloopDir"
-
-wget -O $bloopDir/install.py https://github.com/scalacenter/bloop/releases/download/v$(bloop_version)/install.py \
-&& python $bloopDir/install.py --dest $bloopDir
-
-$bloopDir/bloop ng-stop
-$bloopDir/bloop server &>/dev/null &
+coursier launch ch.epfl.scala:bloopgun-core_2.12:$(bloop_version) -- about
```
Comment (jvican) [inline bin/test.sh]: :tada:

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -3,9 +3,79 @@ on:
   push:
   pull_request:
 jobs:
-  build:
+  windows:
```
Comment (olafurpg) [inline .github/workflows/ci.yml]: What do you think about waiting with this change? It's already a big enough change to move tests from Travis

Comment (tgodzik) [reply .github/workflows/ci.yml]: Sure, I think it's a separate thing. Will try to run those tests on windows myself and debug what;s going on.

File: tests/slow/src/test/scala/tests/mill/MillLspSuite.scala [hunk 3]
```diff
@@ -66,37 +63,6 @@ object MillLspSuite extends BaseImportSuite("mill-import") {
     }
   }
 
-  testAsync("force-command") {
```
Comment (tgodzik) [inline tests/slow/src/test/scala/tests/mill/MillLspSuite.scala]: Those are testing the same things as the sbt tests and it seems Mill tests are less stable.

**Comment by tgodzik** [comment]
@olafurpg This looks promising :D 

>All checks have passed
> 8 successful checks
>

File: .travis.yml [hunk 4]
```diff
@@ -12,25 +12,6 @@ stages:
...
-    - env: TEST="sbt 'slow/testOnly -- tests.maven'"
-      script: bin/test.sh 'slow/testOnly -- tests.maven'
-    - env: TEST="sbt 'slow/testOnly -- tests.feature'"
-      script: bin/test.sh 'slow/testOnly -- tests.feature'
-    - env: TEST="sbt +cross/test"
-      before_script: nvm install 'lts/*' && nvm use 'lts/*'
-      script: sbt +cross/test
     - stage: release
       script:
         - nvm install 'lts/*' && nvm use 'lts/*'
```
Comment (olafurpg) [inline .travis.yml]: BTW We can migrate this job to GH Actions. The latest sbt-ci-release supports GH Actions

**Pull request merged**